design/baremetal-operator/resume-interface.md

```diff
@@ -0,0 +1,90 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# Title
+
+Resume from error state
+
+## Status
+
+provisional
+
+## Summary
+
+This proposal aims to introduce a new declarative API
+to forcibly resume a waiting BareMetalHost resource from an error state
+when a fix is immediately available, so that it could be possible
+to reduce the waiting time required
+
+## Motivation
+
+The current BMO error retry mechanism implements an
+exponential backoff with jittering: this means that when a BMH
+goes into an error state its reconciliation loop gap increases
+as long as the error conditions persists (based on BMH `Status.ErrorCount`).
+
+In some cases it's necessary a human (or more generally an external)
+intervention to fix the issue. If the fix involves amending the
+BMH resource then an immediate reconciliation is triggered and the
+new state is evaluated again.
+Instead, if the fix does not generate a relevant event (for example,
+a firmware update on the host) captured by BMO, then it's necessary
+to wait for the next scheduled reconciliation loop for a new
+state evaluation, and the waiting time could be longer for higher
+values of `Status.ErrorCount`.
+
+### Goals
+
+* A declarative API to force resuming a BMH currently waiting in error state
+
+### Non-Goals
+
+* Clear or remove the current BMH error state
+
+## Proposal
+
+## Design Details
+
+A new annotation `resume.metal3.io` will be used to trigger a new
```

Comment:
What do you think about `retry` instead of `resume`?

Reply:
IMO: retry is fine

Reply:
Sounds fine